### PR TITLE
Initialize operator_versions and upgraders files

### PR DESCRIPTION
**Pull request by tugsbayasgalan**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #66593
* #66592
* #66591

Differential Revision: [D31632601](https://our.internmc.facebook.com/intern/diff/D31632601)

### CODE REVIEW

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31632601).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31632601).

File: torch/jit/operator_upgraders.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+"""
+This is the centralized file for all PyTorch operator upgraders.
+Each function definition here needs to following requirements:
+1. The body of the function must be Torchscriptable
+2. The naming convention of the upgraders should be:
+    <op_name>_<op_overload>_upgrader_<old_version>_<new_version>
+3. The name of the upgrader must be present in operator_versions.yaml
+"""
```
Comment (cccclai) [inline torch/jit/operator_upgraders.py:8]: 4. Add create date and expiration date as well

File: torch/jit/operator_upgraders.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import torch
+import yaml
+from typing import Union
+
+@torch.jit.script
+def div_Tensor_0_3(self: torch.Tensor, other: torch.Tensor) -> torch.Tensor:
+    if (self.is_floating_point() or other.is_floating_point()):
+        return self.true_divide(other)
+    return self.divide(other, rounding_mode='trunc')
```
Comment (cccclai) [inline torch/jit/operator_upgraders.py:17]:
```suggestion
/*
div_Tensor_0_3 is added to for a change in operator in pr xxxxxxx.
Create date: 12/02/2021
Expire date: 06/02/2021
*/
@torch.jit.script
def div_Tensor_0_3(self: torch.Tensor, other: torch.Tensor) -> torch.Tensor:
    if (self.is_floating_point() or other.is_floating_point()):
        return self.true_divide(other)
    return self.divide(other, rounding_mode='trunc')
```

**Pull request closed without merging**